lib/linguist/languages.yml

```diff
@@ -3078,7 +3078,13 @@ Nim:
   color: "#37775b"
   extensions:
   - ".nim"
+  - ".nim.cfg"
+  - ".nimble"
   - ".nimrod"
+  - ".nims"
+  filenames:
+  - config.nims
```

Comment:
There are only [146 files](https://github.com/search?q=filename%3Aconfig.nims) for this filename, so I think we should drop it, especially since these files will already be recognized through the new `.nims` extension.